hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotCache.java

```diff
@@ -178,18 +203,12 @@ public ReferenceCounted<IOmMetadataReader, SnapshotCache> get(String key, boolea
     // when called from SDT (and some) if the snapshot is not active anymore.
     if (!skipActiveCheck && !omSnapshotManager.isSnapshotStatus(key, SNAPSHOT_ACTIVE)) {
       // Ref count was incremented. Need to decrement on exception here.
-      rcOmSnapshot.decrementRefCount();
+      rcOmSnapshot.close();
```

Comment:
This should be `decrementRefCount()` and not `close()`. Some of the background services  for exmaple `SnapshotDeletingService` works on non-Active snapshot. Snapshot is opened by `SnapshotDeletingService`, it will close the currently used snapshot.

Reply:
Had a offline discussion, `close()` internal calls `decrementRefCount()`.
But I think `decrementRefCount()` is better to use from readability perspective otherwise someone will have the same doubt I had.